### ISSUE TITLE
experiment results about qwen2.5VL

### ISSUE THREAD

**Issue by Celina-love-sweet** [yaolinli/TimeChat-Online#25]
Regarding the experimental results on the MLVU dataset in Table 3, the accuracy of Qwen2.5-VL-7B with 1fps input is 66.9%, while the model in the paper has a result of 62.6% with 1fps (100% token). I would like to ask, does this result, after training, show some performance degradation on this dataset, where it is worse than the training results? Also, for Qwen2.5-VL-7B with 1fps input, is this considered normal streaming input, or is it not the type of input where both the video and the question are input at once?

**Comment by yaolinli** [comment]
Thanks for the questions!

(1) On MLVU performance: Yes. After SFT on the open-released Qwen2.5-VL-7B checkpoint, we observe a drop on general offline video benchmarks such as MLVU. Our view is that the base Qwen2.5-VL-7B is already carefully trained and very strong on these tasks; additional SFT without a carefully balanced mix of data types/ratios will hurt offline VideoQA performance. In this work, we intentionally focus on the streaming setting and do not curate training data to optimize MLVU-style VideoQA, so some degradation after SFT is expected.

(2) On the Qwen2.5VL(1 fps) setting in Table 3: The 1 fps result for Qwen2.5-VL-7B is evaluated in the offline mode—i.e., the (1 fps-sampled) video and the question are fed to the model all at once, not streamed.

**Comment by Celina-love-sweet** [comment]
> Thanks for the questions!
> 
> (1) On MLVU performance: Yes. After SFT on the open-released Qwen2.5-VL-7B checkpoint, we observe a drop on general offline video benchmarks such as MLVU. Our view is that the base Qwen2.5-VL-7B is already carefully trained and very strong on these tasks; additional SFT without a carefully balanced mix of data types/ratios will hurt offline VideoQA performance. In this work, we intentionally focus on the streaming setting and do not curate training data to optimize MLVU-style VideoQA, so some degradation after SFT is expected.
> 
> (2) On the Qwen2.5VL(1 fps) setting in Table 3: The 1 fps result for Qwen2.5-VL-7B is evaluated in the offline mode—i.e., the (1 fps-sampled) video and the question are fed to the model all at once, not streamed.

Thank you for your reply. I would like to confirm the following again:

（1）Regarding Table 3, in the third-to-last row of 'Open-Source Offline VideoLLMs', is the result from Qwen2.5VL with a frame rate of 1 frame per second, up to a maximum of 1016 frames, without compression, which gives a result of 66.9 on the MLVU dataset? In the second-to-last row, with a 46.2% reduction, is it using DTD without training, and the result after compressing by 46.2%?

（2）Regarding Table 3, in the third-to-last row of 'Open-source Online VideoLLMs', is it after training but still without reducing tokens (using DTD)? Is it Qwen2.5VL with a frame rate of 1 frame per second, up to a maximum of 1016 frames, giving a result of 62.6% on the MLVU dataset?

（3）If the maximum is 1016 frames, does this mean that for videos exceeding this frame count, they will be uniformly sampled?

**Comment by yaolinli** [comment]
Thanks for the follow-up! Please see the clarifications on Table 3 below:

**1. Open-source Offline VideoLLMs.**
Yes.
Qwen2.5-VL, 1 fps (100%): this is the uncompressed baseline (no token dropping), capped at 1,016 frames, scoring 66.9 on MLVU.
Qwen2.5-VL, 1 fps (46.2%): this applies DTD directly at inference (dropping 46.2% of tokens) without additional training. The same 1,016-frame cap applies.

**2. Open-source Online VideoLLMs.**
Yes.
TimeChat-Online-7B, 1 fps (100%): this is our trained model (Qwen2.5-VL backbone with our SFT data), without token reduction (DTD disabled at inference). The reported MLVU score is 62.6.

**3. Frame Processing.**
When a video has > 1,016 frames, we uniformly sample frames down to the 1,016-frame cap.

Hope this clears things up!

**Comment by Celina-love-sweet** [comment]
> Thanks for the follow-up! Please see the clarifications on Table 3 below:
> 
> **1. Open-source Offline VideoLLMs.** Yes. Qwen2.5-VL, 1 fps (100%): this is the uncompressed baseline (no token dropping), capped at 1,016 frames, scoring 66.9 on MLVU. Qwen2.5-VL, 1 fps (46.2%): this applies DTD directly at inference (dropping 46.2% of tokens) without additional training. The same 1,016-frame cap applies.
> 
> **2. Open-source Online VideoLLMs.** Yes. TimeChat-Online-7B, 1 fps (100%): this is our trained model (Qwen2.5-VL backbone with our SFT data), without token reduction (DTD disabled at inference). The reported MLVU score is 62.6.
> 
> **3. Frame Processing.** When a video has > 1,016 frames, we uniformly sample frames down to the 1,016-frame cap.
> 
> Hope this clears things up!

Thank you for your reply, and I hope to be able to cite your article in the future.
